content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -718,3 +718,60 @@ Clean up the Pod when you're finished with it:
 ```shell
 kubectl delete pod myapp
 ```
+
+### Custom Profile {#custom-profile}
+
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
+
+You can define partial container spec as a custom profile in either YAML or JSON format, and apply it using the `--custom` flag.
```

Comment:
```suggestion
You can define partial container spec as a custom profile in either YAML or JSON format, and
use it to create an ephemeral container based on that profile.
```

Reply:
As [above](https://github.com/kubernetes/website/pull/48154#discussion_r1789653676), custom profile is not limited to the use case of an ephemeral container, I've expressed that more clearly.